packages/epo-react-lib/vite.config.ts

```diff
@@ -5,7 +5,7 @@ import dts from "vite-plugin-dts";
 
 // https://vitejs.dev/config/
 export default defineConfig({
-  plugins: [react(), dts({ insertTypesEntry: true })],
+  plugins: [dts({ insertTypesEntry: true }), react()],
```

Comment:
there was a bunch of errors around the JSX runtime, turns out the definitions plugin needs to run before the React plugin